data/presets/natural/tree.json

```diff
@@ -1,14 +1,20 @@
 {
     "icon": "maki-park",
     "fields": [
+        "genus",
+        "species",
+        "taxon",
         "leaf_type_singular",
         "leaf_cycle_singular",
-        "denotation",
-        "height",
-        "diameter"
```

Comment:
I assume that you removed `diameter` because it is kind of duplicating the `circumference` tag and `diameter` is used less often? In principle, that would be fine for me, but I could also see potential confusion if a mapper finds that the data they entered in the field in the past is suddenly not displayed anymore. Maybe it could still make sense to continue to include the `diameter` field in the `moreFields` section?!

Reply:
Yes It was exactly my goal 😅 So I restore it 👍 Thanks @tyrasd